api/project.godot

```diff
@@ -31,7 +49,7 @@ run/main_scene="res://test/gut_gui.tscn"
 
 [autoload]
 
-PlayAssetPackManager="res://src/play_asset_pack_manager.gd"
+PlayAssetPackManager="*res://src/play_asset_pack_manager.gd"
```

Comment:
Autoload is enabled now since we need to reference enums such as ```PlayAssetPackManager.AssetPackStorageMethod.STORAGE_FILES``` during testing.